### PR TITLE
Corrected the variable name in create/modify Qos

### PR DESCRIPTION
**Pull request by jyotsnalothe**
Please review the changes. 
@sneharai4 

Tested all the create and modify Qos rules automation playbooks on Alletra 9K
[create_qos.txt](https://github.com/HewlettPackard/hpe3par_python_sdk/files/6929122/create_qos.txt)
[modify_qos.txt](https://github.com/HewlettPackard/hpe3par_python_sdk/files/6929123/modify_qos.txt)

### CODE REVIEW

**Comment by jyotsnalothe** [comment]
Updated the test cases for failures, pulled it from Arshad's branch. Also updated the docs for rebranding changes. PFA the UTs results ran after the changes. Please review the changes.
[python_3parclient_unittests.txt](https://github.com/HewlettPackard/hpe3par_python_sdk/files/6952650/python_3parclient_unittests.txt)


@sneharai4

**Comment by sneharai4** [comment]
> Updated the test cases for failures, pulled it from Arshad's branch. Also updated the docs for rebranding changes. PFA the UTs results ran after the changes. Please review the changes.
> [python_3parclient_unittests.txt](https://github.com/HewlettPackard/hpe3par_python_sdk/files/6952650/python_3parclient_unittests.txt)
> 
> @sneharai4

This looks like pythonclient unit test and not sdk. @jyotsnalothe can you please check.

File: test/HPE3ParClient_base.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from testconfig import config
 import datetime
 from functools import wraps
+from hpe3parclient import file_client
```
Comment (sneharai4) [inline test/HPE3ParClient_base.py:26]: Remove the commented line at line number 28

File: README.rst [hunk 2]
```diff
@@ -11,7 +11,10 @@ Requirements
 * Primera OS
 
   * 4.0.0
-* WSAPI service should be enabled on the HPE 3PAR and HPE Primera storage array
+* Alletra OS
+
+  * 9.4.0
```
Comment (sneharai4) [inline README.rst:16]: Change this to 9.3.0

File: README.rst [hunk 3]
```diff
@@ -11,7 +11,10 @@ Requirements
 * Primera OS
 
   * 4.0.0
-* WSAPI service should be enabled on the HPE 3PAR and HPE Primera storage array
+* Alletra OS
```
Comment (sneharai4) [inline README.rst:14]: Change this to Alletra 9000

File: README.rst [hunk 4]
```diff
@@ -11,7 +11,10 @@ Requirements
 * Primera OS
 
   * 4.0.0
```
Comment (sneharai4) [inline README.rst:13]: Change the version to 4.3.1

File: test/HPE3ParClient_base.py (hunk 1 above)
Comment (jyotsnalothe) [reply test/HPE3ParClient_base.py:26]: Addressed rework

File: README.rst (hunk 2 above)
Comment (jyotsnalothe) [reply README.rst]: Addressed rework

File: README.rst (hunk 3 above)
Comment (jyotsnalothe) [reply README.rst]: Addressed rework

File: README.rst (hunk 4 above)
Comment (jyotsnalothe) [reply README.rst]: Addressed rework

**Review by sneharai4 — APPROVED** [review]

**Pull request merged**